tools/gdb/gdb_scripts/wt_debug_script_update.py

```diff
@@ -2,6 +2,8 @@
 import bson
 from pprint import pprint
 
+# FIXME-WT-11881 Update to use gdb.Command like the other python scripts
```

Comment:
I was wondering how you managed to do this without fixing this script. Btw the correct ticket for the fixme is WT-9286. Either that or we have further ticket duplication.

Reply:
I closed out the extra-extra dup